.github/workflows/ci.yml

```diff
@@ -362,12 +362,20 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-latest]
+        os: [ubuntu-latest, macos-latest]
         rust: [stable]
 
     steps:
       - uses: actions/checkout@v2
       - uses: ./
         name: Install Fluvio Local Cluster
       - name: Fluvio command test
-        run: fluvio topic list
+        run: |
+          fluvio version
+          fluvio topic list
+          fluvio topic create "foobar"
+          sleep 3
+          echo foo | fluvio produce "foobar"
+          fluvio consume foobar -o 0 -d
+          # Delete the topic afterword but this looks to not work right now.
+          # fluvio topic delete "foobar"
```

Comment:
I tried this today (with 0.6.0-beta.1) and it didn't work so I'm not sure what's up. Could just be an issue with how bad the github instance is.